pkg/rancher-desktop/assets/lima-config.yaml

```diff
@@ -135,11 +135,14 @@ provision:
     # openresty is backgrounding itself (and writes its own pid file)
     sed -i 's/^command_background/#command_background/' /etc/init.d/openresty
 
-    # configure proxy only when image-allow-list exists
-    ial=/usr/local/openresty/nginx/conf/image-allow-list.conf
+    # configure proxy only when allowed-images exists
+    ial=/usr/local/openresty/nginx/conf/allowed-images.conf
```

Comment:
This isn't `image-allow-list` anymore, should the variable be called `aiconf` or something instead?